core/src/jvmMain/kotlin/com/ustadmobile/core/catalog/contenttype/VideoTypePluginJvm.kt

```diff
@@ -0,0 +1,52 @@
+package com.ustadmobile.core.catalog.contenttype
+
+import com.ustadmobile.core.container.ContainerManager
+import com.ustadmobile.core.container.addEntriesFromZipToContainer
+import com.ustadmobile.core.db.UmAppDatabase
+import com.ustadmobile.lib.db.entities.Container
+import com.ustadmobile.lib.db.entities.ContentEntry
+import com.ustadmobile.lib.db.entities.ContentEntryWithLanguage
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.io.File
+
+class VideoTypePluginJvm: VideoTypePlugin() {
+
+    override suspend fun extractMetadata(filePath: String): ContentEntryWithLanguage? {
+        return withContext(Dispatchers.Default){
+            val file = File(filePath)
+
+            fileExtensions.find { file.name.endsWith(it, true) }
```

Comment:
I'm not sure this is really completely as clear as it can be. It works, but I think it would be clearer to use a normal if statement here. You are not using the return of the fileExtensions.find for anything.

e.g.
```
if(!fileExtensions.any { file.name.endsWith(it, true) }) {
    return@withContext null
}
```